performance_test/script.mjs

```diff
@@ -0,0 +1,98 @@
+import { execSync } from "child_process";
+import { copyFile } from "fs/promises";
+import { performance } from "perf_hooks";
+import { existsSync, mkdirSync, rmdirSync } from "fs";
+
+const DEFAULT_NUMBER_OF_FILE = 5;
+const DEFAULT_ITERATION = 1;
+const BASE_PATH = "./performance_test";
+const DEFAULT_MOCK_FILE_PATH = `${BASE_PATH}/mock_unwrapped_doc.json`;
+const INPUT_MOCK_UNWRAPPED_FILE_FOLDER = `${BASE_PATH}/raw-documents`;
```

Comment:
use path.join, otherwise it wont work on windows

Reply:
done